doc/misc/npm-faq.md

```diff
@@ -67,16 +58,9 @@ programs reference the same copy of it, then use the `npm link` command.
 That's what it's for.  Install it globally, then link it into each
 program that uses it.
 
-## Whatever, I really want the old style 'everything global' style.
```

Comment:
While I agree that this could be worded in a less aggressive style, I do think it's important to stress that npm has a particular design with particular choices, and that those choices are not going to change without serious deliberation. Given the frequency with which people ask if or when npm is going to change this restriction, I feel it's important, for users as well as those of us supporting npm, for some version of this question and answer to remain in the FAQ.